card/config/initializers/02_patches/active_record.rb

```diff
@@ -0,0 +1,66 @@
+# pluck_in_batches:
```

Comment:
no, I needed it in a migration for wikirate and thought it might be useful in wagn, too. Actually,  your comment reminds me that we are using `find_in_batches` somewhere in wagn. `pluck_in_batches` could improve performance there.